banking-app/README.md

```diff
@@ -1,63 +1,63 @@
-    - Example response: Status code 200 with body `{ "balance" : 100 }`
+  - Check balance
+  - It can be called by bank account holders ([users](https://microsoft.github.io/CCF/main/overview/glossary.html#term-Users) in CCF terminology)
+  - Example response: Status code 200 with body `{ "balance" : 100 }`
 - POST /app/transfer/{account_name}
-    - Transfer money from an account to another account
-    - It can be called by bank users ([users](https://microsoft.github.io/CCF/main/overview/glossary.html#term-Users) in CCF terminology)
-    - Example request body: `{ value : 100, user_id_to: 'userA' , account_name_to: 'accountA' }`
-    - Status code for successful calls: 204
+  - Transfer money from an account to another account
+  - It can be called by bank account holders ([users](https://microsoft.github.io/CCF/main/overview/glossary.html#term-Users) in CCF terminology)
+  - Example request body: `{ value : 100, user_id_to: 'userA' , account_name_to: 'accountA' }`
+  - Status code for successful calls: 204
 
 ### Why "Deposit money" API (`POST /app/deposit/{user_id}/{account_name}`) can't be called by users not members?
 
 In real life, it's normal that users can deposit money through ATM at their own will. So it might be unintuitive that users can't call the endpoint. However if we allow users, they could deposit any amount of money with no actual money.
 If you want implement an ATM, the deposit procedure would be:
+
 1. User puts actual money into an ATM.
 2. The ATM counts the money.
 3. The ATM calls "Deposit money" API as bank.
 
-
 ## Why CCF?
 
-There are multiple reasons that you want to use CCF for banking consortium.
+There are multiple reasons that you want to use CCF for a banking consortium.
 
-- The ledger generated by CCF is tamper-proof, it cannot be modified after having been produced. Even users without a full copy of the ledger, holding their transaction receipts, are protected.
+- The ledger generated by CCF is tamper-proof, it cannot be modified after having been produced; it is immutable. Even users without a full copy of the ledger, holding their transaction receipts, are protected.
 - Users can use [transaction receipts](https://microsoft.github.io/CCF/main/audit/receipts.html#receipts) with [application claim](https://microsoft.github.io/CCF/main/use_apps/verify_tx.html#application-claims) as proof of transferring money even without online access to the system (TODO: implement application-claims in the app).
 - Banks don't have to trust each other. A majority of the banks need to agree to make changes on the application with the default [constitution](https://microsoft.github.io/CCF/main/governance/constitution.html#constitution).
 - Banks and users don't have to trust node [operators](https://microsoft.github.io/CCF/main/overview/glossary.html#term-Operators).
 
 ## How to run the app in sandbox
 
 ```bash
-$ cd banking-app
-$ npm i # Necessary only for the first time
+cd banking-app
```

Comment:
I removed the $ so that the cut and paste works. Otherwise you will end up with $ at the start of the command